src/ProcessManagerBundle/Installer.php

```diff
@@ -47,6 +47,15 @@ public function migrateInstall(Schema $schema, Version $version)
         $execTable->addColumn('active', 'boolean')
             ->setDefault(1);
         $execTable->setPrimaryKey(['id']);
+
+        // Permissions
+        $permission = new Permission\Definition();
```

Comment:
not necessary, PM calls all migrations on install.

Reply:
Reverted changes in Installer.php